borg/archiver.py

```diff
@@ -1213,6 +1216,8 @@ def build_parser(self, args=None, prog=None):
                                help='only check last N archives (Default: all)')
         subparser.add_argument('-P', '--prefix', dest='prefix', type=str,
                                help='only consider archive names starting with this prefix')
+        subparser.add_argument('--no-progress', dest='no_progress', action='store_false',
```

Comment:
Yeah, I don't like it either. But the current behavior is to print progress, so the choice was to either change the default, or come up with some option name that was descriptive. --silent/--quiet didn't seem right because the option is just to shut off progress, not -v output.  -v --no-progress is a reasonable combination of options, and -v --silent/--quiet seems silly.

Suggestions?
-- 
Ed Blackman
Sent from my OnePlus One. Please excuse my brevity.


-------- Original Message --------
From: TW <notifications@github.com>
Sent: April 26, 2016 10:40:42 PM EDT
To: borgbackup/borg <borg@noreply.github.com>
Cc: edgewood <ed@edgewood.to>, Mention <mention@noreply.github.com>
Subject: Re: [borgbackup/borg] Print --list output (only) without requiring --info (#865)

> @@ -1213,6 +1216,8 @@ def build_parser(self, args=None, prog=None):
>                                 help='only check last N archives (Default: all)')
>          subparser.add_argument('-P', '--prefix', dest='prefix', type=str,
>                                 help='only consider archive names starting with this prefix')
> +        subparser.add_argument('--no-progress', dest='no_progress', action='store_false',

I'ld prefer to not having --no-whatever options.

Or, said without double negation: I prefer --whatever options. :)

---
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub:
https://github.com/borgbackup/borg/pull/865/files/1f8d5f4103e8b96701fd3fdb72f81a71b18b6125..1901fc0950b336938fa919e34f22452c5f6fe409#r61196387